backend/manager/modules/restapi/types/src/main/java/org/ovirt/engine/api/restapi/types/DiskSnapshotMapper.java

```diff
@@ -14,6 +14,7 @@ public static DiskSnapshot map(org.ovirt.engine.core.common.businessentities.sto
         if (template == null) {
             template = new DiskSnapshot();
         }
+        template.setParent(new DiskSnapshot());
```

Comment:
this means that even a disk snapshot without a parent would have 'parent' set? that's not that great..